### ISSUE TITLE
Add Task to Mamie when Opportunity is Donation - Closed Won

### ISSUE THREAD

**Issue by marthacarolina** [marthacarolina/myApexCode#51]
When an donation is made (closed won) add a task to Mamie, and a notification to Jerry.

**Comment by marthacarolina** [comment]
I added two actions to the Process Builder process named Test 3 that works on opportunities. One action adds a task to Mamie to acknowledge the opportunity, and the second action posts the donation in Jerry's chat roll.

**Comment by marthacarolina** [comment]
the actions added to the process builder were firing when the record was created or updated, so it was over-firing. I deleted the new actions in the process and I added a workflow rule for opportunities when created or updated to meet condition. The workflow rule adds a task to Mamie to acknowledge the opportunity if it hasn't been acknowledge yet. However, I couldn't create an action to post to Jerry. 

I could create a new process to do this, but first I wan't to talk it over with Jerry and see if he wants to add it.
